specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/nova.json

```diff
@@ -0,0 +1,1051 @@
+      "minLength": 1,
+      "maxLength": 50,
+      "pattern": "^[a-zA-Z0-9]+(-[a-zA-Z0-9]+)*"
+    },
+    "dataCenterNameParameter": {
+      "name": "dataCenterName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "description": "Data center name in a managed Cassandra cluster.",
+      "minLength": 1,
+      "maxLength": 50,
+      "pattern": "^[a-zA-Z0-9]+(-[a-zA-Z0-9]+)*"
+    }
+  },
+  "definitions": {
+    "DefaultErrorResponse": {
+      "description": "An error response from the service.",
+      "properties": {
+        "error": {
+          "description": "The error returned from the service.",
+          "$ref": "cosmos-db.json#/definitions/ErrorResponse"
+        }
+      }
+    },
+    "ListClusters": {
+      "description": "List of managed Cassandra clusters and their properties.",
+      "readOnly": true,
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Container for array of clusters.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ClusterResource"
+          }
+        }
+      }
+    },
+    "ClusterResource": {
+      "x-ms-azure-resource": true,
+      "type": "object",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The unique resource identifier of the ARM resource."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the ARM resource.",
+          "minLength": 1,
+          "maxLength": 100
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of Azure resource."
+        },
+        "location": {
+          "type": "string",
+          "description": "The location of the resource group to which the resource belongs."
+        },
+        "tags": {
+          "$ref": "#/definitions/Tags"
+        },
+        "properties": {
+          "type": "object",
+          "description": "Properties of a managed Cassandra cluster.",
+          "properties": {
+            "provisioningState": {
+              "readOnly": true,
+              "description": "(Read only.) The status of the Cosmos DB account at the time the operation was called. The status can be one of following. 'Creating' – the Cosmos DB account is being created. When an account is in Creating state, only properties that are specified as input for the Create Cosmos DB account operation are returned. 'Succeeded' – the Cosmos DB account is active for use. 'Updating' – the Cosmos DB account is being updated. 'Deleting' – the Cosmos DB account is being deleted. 'Failed' – the Cosmos DB account failed creation. 'DeletionFailed' – the Cosmos DB account deletion failed.",
+              "type": "string"
+            },
+            "restoreFromBackup": {
```

Comment:
property name seems to imply a bool. since this is an id, consider restoreFromBackupId